platform/CallChain.h

```diff
@@ -28,11 +28,9 @@ namespace mbed {
 typedef Callback<void()> *pFunctionPointer_t;
 class CallChainLink;
 
-/** \ingroup mbed-os-internal */
-/** \addtogroup platform-internal-api Platform */
-/** @{*/
 /**
  * \defgroup platform_CallChain CallChain class
+ * \ingroup platform-public-api
```

Comment:
See my comment https://github.com/ARMmbed/mbed-os/pull/11073#discussion_r309187311
But happy for @bulislaw  to decide

Reply:
@bulislaw, do you maintain it is public?

Reply:
> Do you mean that it should be public?
At the start of the Epic I asked @kjbracey-arm for a classification of platform APIs betwen public and internal and CallChain was classified as "Private / deprecated (remove?)"
I see this API only used in InterruptManager.h

There's nothing internal about it, it's a generic helper class. It may be true it's not necessary and should be deprecated (let's not think about it here), but I can see someone using it so by making it internal we may "break" peoples code. There's no consequences of making it public (except the usual it should be tested and documented).

Reply:
All right.